### PR TITLE
Responsive

### PR DESCRIPTION
**Pull request by josepplloo**
## Summary

- Layout for small screens added.

## Screenshots

image

## Trello Card

https://trello.com/c/rQsCHUml/15-dise%C3%B1o-responsive

### CODE REVIEW

File: maquetado/scss/tech.scss [hunk 1]
```diff
@@ -30,3 +30,18 @@
     text-align: center;
   }
 }
+
+@media only screen and (max-width: 1024px) {
+  .tech {
+    border-bottom: 2px solid $green;
+    margin: 0 20%;
```
Comment (nmartinelli23) [inline maquetado/scss/tech.scss]: Dont to use %, please.

File: maquetado/scss/tech.scss [hunk 2]
```diff
@@ -30,3 +30,18 @@
...
+
+    &:first-child {
+      border-right: none;
+    }
+
+    &:last-child {
+      border-bottom: none;
+    }
+  }
+}
```
Comment (nmartinelli23) [inline maquetado/scss/tech.scss]: Add a last blanck line at the end.

File: maquetado/index.html [hunk 3]
```diff
@@ -126,7 +126,7 @@ <h6 class="tech-release">
   <footer class="footer">
     <h3 class="footer-title">WE ARE GAME CHANGERS</h4>
     <img class="footer-logo" src="./assets/wolox_logo_white.svg" alt="Wolox Logo">
-    <h6 class="footer-legal">Copirigth 2018. Wolox All rigths reserved</h6>
+    <h6 class="footer-legal">Copyrigth 2018. Wolox All rigths reserved</h6>
```
Comment (nmartinelli23) [inline maquetado/index.html]: Can we change it to 2019 also?

Comment (henryzarza) [inline maquetado/index.html]:
```suggestion
    <h6 class="footer-legal">Copyright 2018. Wolox All rights reserved</h6>
```

File: maquetado/scss/layout.scss [hunk 4]
```diff
@@ -5,3 +5,12 @@
   justify-content: center;
   margin-bottom: 40px;
 }
+
+@media only screen and (max-width: 1024px) {
+  .techs {
+    align-items: center;
+    display: flex;
```
Comment (henryzarza) [inline maquetado/scss/layout.scss]: Delete this. You can replace this by: `grid-template-columns: 1fr`

File: maquetado/scss/layout.scss [hunk 5]
```diff
@@ -5,3 +5,12 @@
...
+    flex-direction: column;
```
Comment (henryzarza) [inline maquetado/scss/layout.scss]: ☝️

File: maquetado/index.html [hunk 6]
```diff
@@ -126,7 +126,7 @@ <h6 class="tech-release">
   <footer class="footer">
     <h3 class="footer-title">WE ARE GAME CHANGERS</h4>
     <img class="footer-logo" src="./assets/wolox_logo_white.svg" alt="Wolox Logo">
-    <h6 class="footer-legal">Copirigth 2018. Wolox All rigths reserved</h6>
+    <h6 class="footer-legal">Copyrigth 2019. Wolox All rigths reserved</h6>
```
Comment (henryzarza) [inline maquetado/index.html]:
```suggestion
    <h6 class="footer-legal">Copyright 2019. Wolox All rights reserved</h6>
```

File: maquetado/scss/navbar.scss [hunk 7]
```diff
@@ -15,3 +15,11 @@
     height: $nav-logo-height;
   }
 }
+
+@media only screen and (max-width: 1024px) {
+  .navbar {
+    &-title {
```
Comment (henryzarza) [inline maquetado/scss/navbar.scss]: Use `nav-title` directly

**Pull request merged**